styles/_section.scss

```diff
@@ -13,6 +13,65 @@
   }
 
   &-subscribe {
-    .dropdown-toggle::after { display: none; }
+    .dropdown-toggle::after {
+      display: none;
+    }
   }
 }
+
+.section-list {
+  margin: 40px 0;
+}
+
+.section-list-item:first-child {
+  border-top: 1px solid $border-color;
+}
+
+.section-list-item {
+  border-bottom: 1px solid $border-color;
+  font-size: $font-size-bigger;
+  padding: 15px 0;
+}
+
+.section-list-item a {
```

Comment:
could some of these be nested to keep the style we use in other `scss` files?

Reply:
Done in https://github.com/zendesk/copenhagen_theme/pull/70/commits/d7a8440c098057ba6d682e8b3a8642a715d58b6d